src/main/java/Wolox/training/models/Book.java

```diff
@@ -0,0 +1,57 @@
+package Wolox.training.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private int pages;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    public Book(String genre, String author, String image, String title, String subtitle, String publisher, String year,
+                int pages, String isbn) {
+        this.genre = genre;
+        this.author = author;
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    public Book(String author, String image, String title, String subtitle, String publisher, String year,
```

Comment:
why are you creating this constructor?

Reply:
I understood from the trello card that the constructor is required by JPA. Isn't it mandatory?

Reply:
the mandatory contructor is the empty one
`public Book() {}`